amundsen_application/static/css/_buttons-default.scss

```diff
@@ -28,6 +28,37 @@
     margin-bottom: 4px;
   }
 
+  &.btn-cancel {
+    border-color: $gray-lighter;
+    font-family: $font-family-sans-serif-bold;
+    font-weight: $font-weight-sans-serif-bold;
+
+    &:focus,
+    &:not(.disabled):hover,
+    &:not([disabled]):hover {
+      background-color: $gray-lighter;
+      border-color: $gray-lighter;
+    }
+  }
+
+  &.btn-icon {
+    border: none;
+    color: $gray-light;
+    -webkit-box-shadow: none !important;
```

Comment:
I don't think we need to specify -webkit for box-shadow. 

https://caniuse.com/#search=box-shadow